.github/workflows/TEMPLATE_provider.yml

```diff
@@ -25,6 +25,8 @@ jobs:
     runs-on: ubuntu-latest
     steps:
     - uses: actions/checkout@v2
+    - name: install protoc
+      run: sudo apt-get install -y protobuf-compiler
```

Comment:
Since we were able to work around protoc, I'd prefer we keep this out of the provider templates

Reply:
done